### DISCUSSION TITLE
How can I automatically update relational columns?

### DISCUSSION THREAD

**Discussion by Doctorthe113** [pocketbase/pocketbase#5813] [Q&A]
Lets say I have a collection called users that has a relational column called `guildsIn`. Another collection called Guilds that has a relational column `usersIn`. `usersIn` keeps track of the users that are in that guild and `guildsIn` keeps track of the guilds a user is in. Now how do i make it so everytime a user enters a new guild aka update their `guildsIn` column, it automatically updates the usersIn column in pokcetbase. I know I can do it from client side but is it possible to do that from server side? I think it's a bit more risky to update Guilds collection from client side. Thanks in advance for suggestions!

**Comment by chnfyi** [comment]
Maybe you can :
1. Have a collection for each guild
2. Create / delete record (with the same `id` as the user `id`) in the related guild(s) each time a user guildsIn field is update.

It's possible to do it server side using a hook like `onRecordAfterUpdateRequest((e) => { // Your code }, "users")`.

**Reply by Doctorthe113** [reply]
Alright I will look into that. Thanks for the suggestion!
